css/index.css

```diff
@@ -107,30 +114,46 @@ header .logo {
 }
 
 .cta .cta-text {
-    font-size: 72px;
+    font-size: 7.2rem;
     text-align: center;
     letter-spacing: 5px;
     display: flex;
     justify-content: center;
     flex-flow: column;
     align-items: center;
-    margin: 0 82px 0 0;
+    margin: 0 9% 0 0; /* 82px/880px = 9.32% */
+}
+
+.cta .cta-text button {
+    border: 1px double black;
```

Comment:
Even though we leave the button width alone, we can change other things using px to be using rems or percentage.